packages/react-composites/src/composites/CallComposite/styles/CallScreen.styles.ts

```diff
@@ -30,27 +30,22 @@ export const callControlsContainer = mergeStyles({
 export const containerStyles: IStackStyles = {
   root: {
     height: '100%',
-    width: '100%',
-    flexDirection: 'column',
-    display: 'flex'
+    width: '100%'
   }
 };
 
 export const subContainerStyles: IStackStyles = {
   root: {
     overflow: 'hidden',
     width: '100%',
-    flexDirection: 'column',
-    display: 'flex',
-    flexBasis: '0'
```

Comment:
Not sure if I remember this right, I remember set this flexBasis is something related to safari browser(it has the worst flex box support among all the browsers) compatibility, can we double check all the stylings are correct in safari/firefox browser?

Reply:
Good note, I will double check this

Reply:
Yep looks to be good on safari
Removing fixed height:
![image](https://user-images.githubusercontent.com/2684369/126701625-87d8a8eb-12ed-46ab-959c-a9110f4c6860.png)
default experience:
![image](https://user-images.githubusercontent.com/2684369/126701645-32eb667b-67ad-4681-9ce8-124651865812.png)

Reply:
And firefox: 
![image](https://user-images.githubusercontent.com/2684369/126704567-bc9cdbac-a4b1-4d99-9d5d-1082b1975df0.png)